src/pages/publication/contributors_tab/components/SortableTable.tsx

```diff
@@ -98,12 +104,13 @@ interface SortableTableProps {
 }
 
 const SortableTable: FC<SortableTableProps> = ({ listOfContributors, push, remove, swap }) => {
-  const { setFieldValue }: FormikProps<Publication> = useFormikContext();
+  const { setFieldValue, values }: FormikProps<Publication> = useFormikContext();
 
   const handleOnSortEnd = ({ oldIndex, newIndex }: any) => {
-    setFieldValue(`${ContributorFieldNames.CONTRIBUTORS}[${oldIndex}].sequence`, newIndex);
-    setFieldValue(`${ContributorFieldNames.CONTRIBUTORS}[${newIndex}].sequence`, oldIndex);
     swap(oldIndex, newIndex);
+    for (let index in values.entityDescription.contributors) {
+      setFieldValue(`${ContributorFieldNames.CONTRIBUTORS}[${index}].sequence`, +index);
```

Comment:
bare jeg som trodde at index ble objektet heller enn indeksen i den type for-loop? :p 
og hva betyr +index?

Reply:
ja hæ: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in
what am I missing?

Reply:
det er for in, ikke for of eller foreach 😁 
https://thecodebarbarian.com/for-vs-for-each-vs-for-in-vs-for-of-in-javascript

Reply:
+index betyr at det castes til et nummer